.circleci/config.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           root: .
           paths:
             - node_modules
-            - packages/*/node_modules
```

Comment:
We have changed the file structure packages will be inside package/**/*
Examples : 
packages/contact-center/node_modules
packages/widgets/node_modules (https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-583602)